Extend Note for PostAllAssetLinksById (#422)
admin-shell-io/aas-specs-api: documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces.adoc

====

> Note: The SpecificAssetId/name “_globalAssetId_” is defined in Contraint AASd-116 in IDTA-0001.

Reviewer: ```suggestion
Note:  “_globalAssetId_”  is a predefined name (see  Constraint AASd-116 in IDTA-0001).
```

> Its value shall be used as the value for the _AssetInformation/globalAssetId_ attribute of the associated AAS.

Reviewer: ```suggestion
If a specific asset ID uses this predefined name "_globalAssetId_" then its value shall be used as the value for the _AssetInformation/globalAssetId_ attribute of the associated AAS.
```

It shall not be treated as an additional item of _AssetInformation/specificAssetIds_.

> Furthermore, a potentially included SpecificAssetId/externalSubjectId attribute shall be ignored, if and only if the SpecificAssetId/name is equal to “_globalAssetId_”.

Reviewer: ```suggestion
Furthermore, a potentially included SpecificAssetId/externalSubjectId attribute for a specific asset ID with name "globalAssetId" shall be considered an invalid specific asset ID.
```
